### ISSUE TITLE
To Discuss: Change when new posts appear (Timing)

### ISSUE THREAD

**Issue by TilmanDW** [DW-ReCo/kid-game-framework#125]
I was working on verifying a post. In the meantime the timeline moved on and many other posts expired. So when I was back in the timeline, I could only scroll through posts, but no more interaction was possible. 

But at the same time no new content was loading. So I had nothing to do but wait for a new post. We should try to avoid that by maybe offering new content, when all other content has been "worked on" (=blocked, retweeted, run out of time)

**Comment by nikonikoniko** [comment]
Why was there no new content loading?  Had everything already loaded?

On one hand, looking through a bunch of posts that you failed is kind of "the point", as it motivates you with stress to actually get them done quickly enough to get all of them done.

On the other hand - the timing is really important to actually make it playable and fun.

**Comment by TilmanDW** [comment]
I agree, people should read through the posts. But I'm sure some people will go over them, seeing that everything expired and will want to "move on" - and it there is nothing else coming, it's dissapointing. 

We will of course reach at one point the end of the content, but if that's not yet the case (and there weren'tthat many posts there yet) it should move on, I think.

**Comment by nikonikoniko** [comment]
I will try to think of some solutions, but this might be a little hard to do :)

**Comment by TilmanDW** [comment]
We can also discuss this in the meeting again - maybe try it together to see if we run into the same issue and tehen frind a solution for it.
